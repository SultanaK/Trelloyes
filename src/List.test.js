import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import List from './List';

describe('<List />', () => {
    it('renders without crashing', () => {
        // Render the component, as JSON
        const tree = renderer.create( < List header = ''
                allCards = {{}}
                cardIds = {[]}/ > ).toJSON();
        // Check whether it matches the previous snapshot
        // Stored in __snapshots__/App.test.js.snap
        expect(tree).toMatchSnapshot();
    });
});