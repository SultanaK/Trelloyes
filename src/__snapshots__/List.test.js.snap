// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<List /> renders without crashing 1`] = `
<section
  className="List"
>
  <header
    className="List-header"
  >
    <h2>
       
      
       
    </h2>
  </header>
  <div
    className="List-Cards"
  >
    <button
      className="List-add-button"
      type="button"
    >
      +Add Random Card
    </button>
  </div>
</section>
`;
